models/demos/segformer/tt/ttnn_segformer_efficient_selfattention.py

```diff
@@ -83,15 +83,15 @@ def __call__(
                 strategy=mm_a_x_strategy,
                 orientation=ttnn.ShardOrientation.ROW_MAJOR,
             ),
-            dtype=ttnn.bfloat8_b,
+            dtype=ttnn.bfloat16,
         )
         query = ttnn.linear(
             hidden_states,
             parameters.query.weight,
             bias=parameters.query.bias,
             memory_config=mm_a_x_memory_config,
             core_grid=ttnn.CoreGrid(y=mm_a_y, x=mm_a_x),
-            dtype=ttnn.bfloat8_b,
+            dtype=ttnn.bfloat16,
```

Comment:
curious why we had to change the datatype here?

Reply:
bfloat8_t forces you use TILE_Layout. and since we are looking to use all cores I needed bfloat16.